pkg/kubernetes_clusters.go

```diff
@@ -0,0 +1,16 @@
+package clouddriver
+
+type kubernetes_clusters struct {
```

Comment:
In Go, we [export our structs](https://tour.golang.org/basics/3) that need to be visible outside their package. Additionally, all fields use Pascal case.

Rename this to `Cluster`.